### PR TITLE
update quick start and feature list in README

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Signed-off-by: FeynmanZhou <feynman@kubesphere.io>

### What type of PR is this?

/kind documentation

### What this PR does / why we need it:

Replace the All-on-one K8s-based installation with K3s-based installation for KubeSphere quick start. The K3s-based installation process is much faster and easier than the K8s-based installation. 

### Which issue(s) this PR fixes:

See https://github.com/kubesphere/kubesphere/issues/4468

### Special notes for reviewers:
```
None
```

### Does this PR introduced a user-facing change?

```release-note
None
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,30 +119,38 @@ KubeSphere uses a loosely-coupled architecture that separates the [frontend](htt
 
 ## Latest release
 
-🎉 KubeSphere 3.2.0 is now available! See the [Release Notes For 3.2.0](https://kubesphere.io/docs/release/release-v320/) for the updates.
-
+🎉 KubeSphere 3.2.0 is now available! It has supported up to Kubernetes 1.22, see the [Release Notes For 3.2.0](https://kubesphere.io/docs/release/release-v320/) for the updates.
```
Comment (Patrick-LuoYu) [inline README.md:122]: It has supported up to Kubernetes 1.22, see the -> The latest Kubernetes version 1.22 is now supported. See the

File: README.md [hunk 2]
```diff
@@ -113,30 +119,38 @@ KubeSphere uses a loosely-coupled architecture that separates the [frontend](htt
...
 
 ```yaml
 kubectl apply -f https://github.com/kubesphere/ks-installer/releases/download/v3.2.0/kubesphere-installer.yaml
    
 kubectl apply -f https://github.com/kubesphere/ks-installer/releases/download/v3.2.0/cluster-configuration.yaml
 ```
+#### All-in-one
 
-2. You can run the following command to view the installation logs. After KubeSphere is successfully installed, you can use `http://IP:30880` to access the KubeSphere Console with the default account and password (admin/P@88w0rd).
+👨‍💻 No Kubernetes? You can install a single-node Kubernetes/K3s and KubeSphere on your Linux machine with [KubeKey](https://github.com/kubesphere/kubekey). Let's take K3s as an example:
```
Comment (Patrick-LuoYu) [inline README.md:138]: You can use [KubeKey](https://github.com/kubesphere/kubekey) to install both KubeSphere and Kubernetes/K3s in single-node mode on your Linux machine.

File: README.md (hunk 1 above)
Comment (zryfish) [reply README.md:122]: Actually, there are some known issues with v1.22, we used the word `experimental` in ks-installer https://github.com/kubesphere/ks-installer#prerequisites .

File: README.md [hunk 3]
```diff
@@ -113,30 +119,38 @@ KubeSphere uses a loosely-coupled architecture that separates the [frontend](htt
...
+# Download KubeKey
+curl -sfL https://get-kk.kubesphere.io | VERSION=v1.2.0 sh -
+# Make kk executable
+chmod +x kk
+# Create a cluster
+./kk create cluster --with-kubernetes v1.21.4-k3s --with-kubesphere v3.2.0
 ```
 
-> 👨‍💻 No Kubernetes cluster? Try [All-in-one](https://kubesphere.io/docs/quick-start/all-in-one-on-linux/) to install a single-node Kubernetes and KubeSphere on your Linux machine. 
+You can run the following command to view the installation logs. After KubeSphere is successfully installed, you can use `http://IP:30880` to access the KubeSphere Console with the default account and password (admin/P@88w0rd).
```
Comment (Patrick-LuoYu) [inline README.md:149]: you can access the KubeSphere web console at `http://IP:30880` and log in using the default administrator account (admin/P@88word).

**Review by Patrick-LuoYu — COMMENTED** [review]
Please see the comments. Thanks!

File: README.md (hunk 3 above)
Comment (FeynmanZhou) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (FeynmanZhou) [reply README.md]: Done.

**Comment by FeynmanZhou** [comment]
> Please see the comments. Thanks!

@Patrick-LuoYu @zryfish 
Updated. Please help to review it.

**Pull request merged**